src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/CohortsSummaryViewModelMapper.cs

```diff
@@ -43,7 +43,7 @@ public async Task<CohortsViewModel> Map(CohortsByAccountRequest source)
                     "with transfer sending employers",
                     _linkGenerator.CommitmentsV2Link($"{source.AccountHashedId}/unapproved/with-transfer-sender")),
                 
-                BackLink = _linkGenerator.CommitmentsLink($"accounts/{source.AccountHashedId}/apprentices/home")
+                BackLink = _linkGenerator.CommitmentsV2Link($"{source.AccountHashedId}/unapproved")
```

Comment:
I think Url.Action is better in this scenario, but Url may not be available inside the mapper, may need to populate it in the controller. 
It is a suggestion, I will leave it up to you to decide, whether it is just too much work for this.